packages/manager/src/features/Linodes/LinodesCreate/SelectPlanPanel/PlanContainer.tsx

```diff
@@ -102,7 +102,9 @@ export const PlanContainer = ({
                   return (
                     <TableCell
                       center={center}
-                      className={classes.headerCell}
+                      className={`${
+                        cellName === 'Plan' ? 'planHeaderCell' : ''
+                      } ${classes.headerCell}`}
```

Comment:
Can we use `cx` here? Also, it would be nice if `planHeaderCell` wasn't just a string class name

```suggestion
                      className={cx(
                        { planHeaderCell: cellName === 'Plan' },
                        classes.headerCell
                      )}
```